tests/multiprocessing_smoke_tests.py

```diff
@@ -19,6 +19,11 @@
         (LearningProgressCurriculum, create_nethack_env, (nethack_env.task_space,), {}),
         (PrioritizedLevelReplay, create_nethack_env, (nethack_env.task_space,), {"device": "cpu", "suppress_usage_warnings": True, "num_processes": N_ENVS}),
         (SimpleBoxCurriculum, create_cartpole_env, (cartpole_env.task_space,), {}),
+        (AnnealingBoxCurriculum, create_cartpole_env, (cartpole_env.task_space,), {
+            'start_values': [0.2, 0.5],
+            'end_values': [0.8, 0.9],
+            'steps': 10
+        }),
```

Comment:
Have you tried running this? Does it work? Also please try to add some more thorough unit tests in a separate file (check that the annealed values are what you would expect after a specific number of steps).

Reply:
Hi, should I have to create a custom test using unittest with a different file??

Reply:
Yes, it would be great if you could create a specific test file for simulated annealing at write some common sense tests to make sure the internals work as expected.